From 09d03f7cc1a6653b47f1c13a9eef4cfb0cb0369a Mon Sep 17 00:00:00 2001
From: Ray Speth <speth@mit.edu>
Date: Wed, 16 Aug 2023 16:39:40 -0400
Subject: [PATCH] Relax tolerance on test_phase_order_surf_jacobian

This test was failing by a slim margin on ppc64le Linux builds
---
 test/python/test_reactor.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/python/test_reactor.py b/test/python/test_reactor.py
index 4bb8d37f7..cfaa63ca8 100644
--- a/test/python/test_reactor.py
+++ b/test/python/test_reactor.py
@@ -1370,7 +1370,7 @@ class TestReactorJacobians(utilities.CanteraTest):
         net.derivative_settings = {"skip-coverage-dependence":True}
         net.initialize()
         # check that they two arrays are the same
-        self.assertArrayNear(r1.jacobian, r2.jacobian, 1e-6, 1e-8)
+        self.assertArrayNear(r1.jacobian, r2.jacobian, 2e-6, 1e-8)
 
 # A rate type used for testing integrator error handling
 class FailRateData(ct.ExtensibleRateData):
-- 
2.39.2 (Apple Git-143)

